### PR TITLE
added deploy bash history to logs

### PR DESCRIPTION
**Pull request by SudoNikon**

### CODE REVIEW

**Comment by SudoNikon** [comment]
I know this may need some work, if so I want to work through it with you.

File: deploy/definitions/opsworks_deploy_user.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    uid next_free_uid
+    gid deploy[:group]
+    home deploy[:home]
+    supports :manage_home => true
+    shell deploy[:shell]
+    not_if do
+      existing_usernames = []
+      Etc.passwd {|user| existing_usernames << user['name']}
+      existing_usernames.include?(deploy[:user])
+    file "/home/deploy/.bashrc" do
```
Comment (mshepet) [inline deploy/definitions/opsworks_deploy_user.rb]: this file resource is inside the "not_if do" argument inside the user resource.  This file resource should be its own resource separate from the user resource.

**Comment by SudoNikon** [comment]
The point of this PR:
We are starting to track bash history of Deploy user.  We are doing so in an effort to monitor that Deploy user is not manually running GIT commands.

This PR will override the .bashrc profile for the Deploy user and force it to write history to bash_history during the session, were previously it would wait until the session was closed prior to writing to the file.

**Comment by mshepet** [comment]
👍

**Pull request merged**